### ISSUE TITLE
Implementing receiving MIDI

### ISSUE THREAD

**Issue by Boscop** [heartscrytech/DigisparkMIDI#1]
I'm interested in implementing the ability to receive MIDI via USB.
How hard would it be? I'm very familiar with the MIDI standard but new to software USB.
Is there some code that already does it that just needs to be ported?

**Comment by Harvie** [comment]
Yes. This would be so great! I would love to have bidirectional usb-midi converter based on this.

**Comment by CaptainCredible** [comment]
check out cocomake7, more comes with working USBmidi library.

**Comment by Harvie** [comment]
@CaptainCredible any link to that library? can't find it.

**Comment by CaptainCredible** [comment]
It's a board with a bunch of libraries, i think it is cross compatible with the digispark. cant remember if the reset pin is disabled or not on digispark. On the cocomake7 the reset pin is used as gpio pin.

https://cocomake7.github.io/?
